samples/matter/thermostat/src/temperature_manager.cpp

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+
+#include "temperature_manager.h"
+#include "app_config.h"
+#include "app_task.h"
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+using namespace chip;
+using namespace ::chip::DeviceLayer;
+
+constexpr EndpointId kThermostatEndpoint = 1;
+
+namespace ThermostatAttributes = chip::app::Clusters::Thermostat::Attributes;
+
+uint8_t ReturnCompleteValue(int16_t Value)
+{
+    uint8_t result = Value/100; // Convert and return only complete part of value to printable type 
+    return result;
+
+}
+
+uint8_t ReturnReminderValue(int16_t Value)
+{
+
+    uint8_t remainder = (Value%100 + 5)/10; // Converts and returns only reminder part of value to printable type. 
```

Comment:
```suggestion
    uint8_t reminder = (Value%100 + 5)/10; // Converts and returns only reminder part of value to printable type. 
```

Reply:
I meant remainder value in this situation. But it might be not adequate name for this value

Reply:
Oh, I see :smile: